src/containers/header/header.jsx

```diff
@@ -136,7 +136,7 @@ const Header = ({ isMenuOpen, showMenu, hideMenu, height, user, theme }) => (
             }
         </MenuHeader>
         {isMenuOpen && (
-            <MenuBar>
+            <MenuBar onClick={isPhoneResolution ? hideMenu : null}>
```

Comment:
Also I think it's better to implement hide using CSS. It will provide us with ability to add hide/show animations.